extensions/amp-story/1.0/amp-story-quiz.css

```diff
@@ -0,0 +1,122 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins&display=swap');
```

Comment:
`!important` everything that's not meant to be overridden

Reply:
Done

Reply:
This font only loads latin characters. We need something for other alphabets.

Reply:
Opened #25716 to track this